src/tools/annotation2proto.ts

```diff
@@ -0,0 +1,22 @@
+/**
```

Comment:
Probably worth creating an `annotation2proto-test.ts` file. It'll be a really really simple test, but it's nice to have it there in case this file ever gets more complicated in the future.

Reply:
good point, added.